maven-java/kalix-maven-archetype-event-sourced-entity/src/main/resources/archetype-resources/.env

```diff
@@ -0,0 +1,11 @@
+# this is the port where the kalix runtime container will be exposed
+# when running multiple services on your local machine, make sure that the ports unique
+ADVERTISED_HTTP_PORT=9000
+
+# this is the port where the user services (your applicaiton) will open
+# when running multiple services on your local machine, make sure that the ports unique
+USER_SERVICE_PORT=8080
+
+# this variable defines the host where the kalix runtime (running in docker) 
+# will reach the user service in local development
+USER_SERVICE_HOST=host.docker.internal
```

Comment:
This property is in most case irrelevant. Users can't simply change it. Maybe if using something like Colima or similar.